.circleci/config.yml

```diff
@@ -2,7 +2,7 @@ version: 2
 jobs:
   test:
     docker:
-      - image: circleci/node:4-browsers
+      - image: circleci/node:8-browsers
```

Comment:
Had to upgrade to 8 for `lint-staged`, see [CI Job#14](https://circleci.com/gh/fathyb/analytics.js-core/14)

Reply:
The final artifact is built by the [analytics.js repo](https://github.com/segmentio/analytics.js) so it should only affect tests